### ISSUE TITLE
Tensorflow Object Detection API Detects Old Classes as My Custom Object

### ISSUE THREAD

**Issue by mstferis** [tensorflow/models#3427]
Please go to Stack Overflow for help and support:

http://stackoverflow.com/questions/tagged/tensorflow

Also, please understand that many of the models included in this repository are experimental and research-style code. If you open a GitHub issue, here is our policy:

1. It must be a bug, a feature request, or a significant problem with documentation (for small docs fixes please send a PR instead).
2. The form below must be filled out.

**Here's why we have that policy**: TensorFlow developers respond to issues. We want to focus on work that benefits the whole community, e.g., fixing bugs and adding features. Support only helps individuals. GitHub also notifies thousands of people when issues are filed. We want them to see you communicating an interesting problem, rather than being redirected to Stack Overflow.

------------------------

### System information
- **What is the top-level directory of the model you are using**:
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:Linux Ubuntu 16.04
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**:1.3.0
- **Bazel version (if compiling from source)**:-
- **CUDA/cuDNN version**:8/6
- **GPU model and memory**:NVidia Quadro M4000

You can collect some of this information using our environment capture script:

https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh

You can obtain the TensorFlow version with

python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"

### Describe the problem
I am triying to create a custom object detector with Tensorflow Object Detection API. I followed the API documentation,[ raccoonn detector tutorial](https://towardsdatascience.com/how-to-train-your-own-object-detector-with-tensorflows-object-detector-api-bec72ecfe1d9), [macncheese tutorial](https://pythonprogramming.net/introduction-use-tensorflow-object-detection-api-tutorial/) and many others. I am using pre-trained faster rcnn model and default config file values (i edited "PATH TO BE CONFİGURED" areas correctly)(my label map has only one item and its id is 1). After training with 6660 training examples (tested at approximate 1500 and 5000 steps), results are not perfect but its okay for my test images. The main problem is when i test the model with different images that doesn't contain my custom object, it detects other objects as my custom object. Especially, if the object is in coco dataset (my model pre-trained on coco dataset) like car, knife, laptop etc. model detecting their location and tagging them as my custom object. I tried SSD mobilenet too and same issue. Is this a bug or am i missing something ? 

### Source code / logs
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem.

**Comment by bignamehyp** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
